src/app/sections/intro-section.tsx

```diff
@@ -3,22 +3,24 @@ import Image from "next/image";
 
 export default function IntroSection() {
   return (
-    <section className="flex lg:flex-row flex-col gap-8 justify-center items-center py-16 px-8 bg-white">
-      <div className="flex flex-col lg:w-1/3">
-        <h2 className="text-4xl font-medium text-center lg:text-start">
-          {intro.headline}
-        </h2>
-        <p className="text-xl mt-8 lg:w-[80%] text-center lg:text-start">
-          {intro.body}
-        </p>
+    <section className="bg-white">
+      <div className="flex flex-col lg:flex-row gap-8 items-center justify-center max-w-7xl py-16 px-8 m-auto">
+        <div className="flex flex-col lg:w-1/3">
```

Comment:
The width is too small compared to the feedback section. Maybe remove `justify-center`, this is doing nothing in mobile breakpoint, and it's making blank space on the sides for large screens.

And consider removing `lg:w-1/3` , let the text expands to the remaining space.

![Screenshot 2025-03-22 at 9 31 37 PM](https://github.com/user-attachments/assets/f00e74e6-34b8-41a4-8d3f-6af5d0955597)


```suggestion
      <div className="flex flex-col lg:flex-row gap-8 items-center max-w-7xl py-16 px-8 m-auto">
        <div className="flex flex-col">
```